src/component/util.js

```diff
@@ -0,0 +1,13 @@
+function makeDom(elementName, innerHTML) {
+  if (!innerHTML) {
+    return { dom: document.createElement(elementName) };
+  }
+  const dom = document.createElement(elementName);
+  dom.innerHTML = innerHTML;
+
+  return {
+    dom,
+  };
+}
+
+export { makeDom };
```

Comment:
```suggestion
export function makeDom(elementName, innerHTML) {
  if (!innerHTML) {
    return { dom: document.createElement(elementName) };
  }
  const dom = document.createElement(elementName);
  dom.innerHTML = innerHTML;

  return {
    dom,
  };
}
```
함수에서 바로 export를 해도 되지 않을까요~?

Reply:
수정했습니다.